simplq/src/services/notifications.js

```diff
@@ -0,0 +1,40 @@
+function checkNotificationPromise() {
```

Comment:
Will use this file as boilerplate for notif permissions later.

For now, submitting PR to prevent warnings in useEffect

Reply:
Could you take care of these warnings before merging please?

Reply:
? These warnings have been removed through useCallback

Reply:
Yeah, all good now. But make sure to replace the materialUI buttons with the ones we made in the common folder later on.

Reply:
And try to make do without Typography, since it's mixing styles with components which we'd prefer not to do.

Reply:
Essentially only use external components like materialUI if it provides some functionality that's difficult to replicate without it. That's why we've kept the old stepper component since it's hard to make and even the textField, since it handles error reporting by itself.